### ISSUE TITLE
Identification of the people who create records

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/monarch-phenote#25]
We want to be able most easily create proper attribution for annotations or Phenopacket creation. Easiest way to ensure this might be to enable login via ORCID. 

IF NOT, other mechanisms for a diversity of person identifiers should be supported, such as a foaf page or a OCLC id.

ALSO, more than one person ID needs to be allowed for each record or phenopacket

**Comment by cmungall** [comment]
What underlying property should be used for associating an association 
to a person? dc:creator?

We can shorten this to whatever we like in the yaml

**Comment by mellybelly** [comment]
I think dc:creator is good

**Comment by jmcmurry** [comment]
Not to overcomplicate it, but do we want to capture any of the semi-automation that may be the starting point for the creation? Eg. text-mining pipelines.
